specs/derivation.md

```diff
@@ -798,24 +798,41 @@ To interact with the engine, the [execution engine API][exec-engine] is used, wi
 
 [exec-engine]: exec-engine.md
 
-- [`engine_forkchoiceUpdatedV2`] — updates the forkchoice (i.e. the chain head) to `headBlockHash` if different, and
+- [`engine_forkchoiceUpdatedV3`] — updates the forkchoice (i.e. the chain head) to `headBlockHash` if different, and
   instructs the engine to start building an execution payload if the payload attributes parameter is not `null`.
-- [`engine_getPayloadV2`] — retrieves a previously requested execution payload build.
-- [`engine_newPayloadV2`] — executes an execution payload to create a block.
+- [`engine_getPayloadV3`] — retrieves a previously requested execution payload build.
+- `engine_newPayload`
+  - [`engine_newPayloadV2`] — executes a pre-Ecotone execution payload to create a block.
+  - [`engine_newPayloadV3`] — executes a post-Ectone execution payload to create a block.
 
-The current version of `op-node` uses the `v2` RPC methods from the engine API, whereas prior versions used the `v1`
-equivalents. The `v2` methods are backwards compatible with `v1` payloads but support Shanghai.
+The current version of `op-node` uses the `v3` Engine API RPC methods as well as `engine_newPayloadV2`.
+`engine_newPayloadV3` can only be called with an Ecotone execution payload. Both `engine_forkchoiceUpdatedV3` and
+`engine_getPayloadV3` are backwards compatible with pre-Ecotone payloads.
 
-[`engine_forkchoiceUpdatedV2`]: exec-engine.md#engine_forkchoiceupdatedv2
-[`engine_getPayloadV2`]: exec-engine.md#engine_getpayloadv2
+Prior versions of `op-node` used `v2` and `v1` methods.
+
+[`engine_forkchoiceUpdatedV3`]: exec-engine.md#engine_forkchoiceupdatedv3
+[`engine_getPayloadV3`]: exec-engine.md#engine_getpayloadv3
 [`engine_newPayloadV2`]: exec-engine.md#engine_newpayloadv2
+[`engine_newPayloadV3`]: exec-engine.md#engine_newpayloadv3
+
+The execution payload is an object of type [`ExecutionPayloadV3`][eth-payload].
 
-The execution payload is an object of type [`ExecutionPayloadV2`][eth-payload].
+[eth-payload]: https://github.com/ethereum/execution-apis/blob/main/src/engine/cancun.md
 
-[eth-payload]: https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#payloadattributesv2
+The `ExecutionPayload` has the following requirements:
 
```

Comment:
I am a bit confused here what the difference between post canyon and pre ecotone means.

Perhaps we can do
- Bedrock
- Canyon
- Ecotone